packages/commonwealth/client/styles/components/sidebar/collapsable_button.scss

```diff
@@ -0,0 +1,64 @@
+// This is an invisible box that allows us to hover over it in order to make the collapsable button visible
+.hover-box {
```

Comment:
we should always encapsulate all styles in the file with parent class, so for example for component CollapsableSidebarButtonUnscoped, the file should be named `CollapsableSidebarButtonUnscoped.scss` and the parent class should be `CollapsableSidebarButtonUnscoped` and then inside every descendant classname. 

```
.CollapsableSidebarButtonUnscoped {
   .other {}
   . classes {}
}
```

This is very important for style encapsulation